flow/layers/physical_shape_layer_unittests.cc

```diff
@@ -174,8 +174,8 @@ TEST_F(PhysicalShapeLayerTest, ElevationComplex) {
     // there.
     EXPECT_EQ(layers[i]->paint_bounds(),
               (PhysicalShapeLayer::ComputeShadowBounds(
-                  layer_path.getBounds(), initial_elevations[i],
-                  1.0f /* pixel_ratio */)));
+                  layer_path, initial_elevations[i], 1.0f /* pixel_ratio */,
+                  SkMatrix())));
```

Comment:
I'd like to see some new tests written that verify that we produce the expected bounds for a given set of parameters.

Reply:
Yes, testing with different values of `dpr` and `ctm` will be nice. Working on it.